pom.xml

```diff
@@ -22,7 +22,7 @@
     <tag>HEAD</tag>
   </scm>
   <properties>
-    <quarkus.version>3.15.4</quarkus.version>
+    <quarkus.version>3.22.0</quarkus.version>
```

Comment:
Can we fix this without bumping Quarkus?

Reply:
Sure, the `Quarkus.getStatus()` just be added in `3.22.0`.